pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/NonPepperKitServiceTest.java

```diff
@@ -0,0 +1,77 @@
+package org.broadinstitute.dsm.juniperkits;
+
+import static junit.framework.TestCase.assertEquals;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.TimeZone;
+
+import org.broadinstitute.dsm.db.dto.user.UserDto;
+import org.broadinstitute.dsm.model.nonpepperkit.NonPepperStatusKitService;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class NonPepperKitServiceTest {
+    NonPepperStatusKitService nonPepperStatusKitService = new NonPepperStatusKitService();
+
+    @Test
+    public void getUserEmailForFieldsTest() {
+        UserDto userDto1 = new UserDto(1, "user1", "user1Email", null, 1);
+        UserDto userDto2 = new UserDto(2, "user2", "user2Email", null, 1);
+        UserDto userDto3 = new UserDto(3, "user3", "user3Email", null, 1);
+        UserDto userDto4 = new UserDto(4, "user4", "user4Email", null, 1);
+        UserDto userDto5 = new UserDto(5, "user5", "user5Email", null, 1);
+
+        HashMap<Integer, UserDto> map = new HashMap<>();
+        map.put(1, userDto1);
+        map.put(2, userDto2);
+        map.put(3, userDto3);
+        map.put(4, userDto4);
+        map.put(5, userDto5);
+
+        String user1TestEmail = nonPepperStatusKitService.getUserEmailForFields("1", map);
+        Assert.assertEquals(user1TestEmail, "user1Email");
+
+        String userNewTestEmail = nonPepperStatusKitService.getUserEmailForFields("user6Email", map);
+        Assert.assertEquals(userNewTestEmail, "user6Email");
+
+        String user7TestEmail = nonPepperStatusKitService.getUserEmailForFields("7", map);
+        Assert.assertEquals(user7TestEmail, "DSM User");
+
+        Assert.assertEquals("", nonPepperStatusKitService.getUserEmailForFields(null, map));
+
+    }
+
+    @Test
+    public void convertTimeStringIntoTimeStampTest() {
```

Comment:
I'm not sure this test is necessary, or could be greatly reduced. The convertTimeStringIntoTimeStamp method checks input then calls a library function, so the only thing we ned to test is whether bad input is handled correctly.

Reply:
I am mostly trying to make sure the parsed Date from `nonPepperStatusKitService.convertTimeStringIntoTimeStamp` has the required format